osu.Game.Rulesets.Osu/Mods/OsuModAimAssist.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Objects.Types;
+using osu.Game.Rulesets.Osu.Objects.Drawables;
+using osu.Game.Rulesets.UI;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModAimAssist : Mod, IApplicableToDrawableHitObjects, IUpdatableByPlayfield
+    {
+        public override string Name => "AimAssist";
+        public override string Acronym => "AA";
+        public override IconUsage Icon => FontAwesome.Solid.MousePointer;
+        public override ModType Type => ModType.Fun;
+        public override string Description => "No need to chase the circle, the circle chases you";
+        public override double ScoreMultiplier => 1;
+        public override Type[] IncompatibleMods => new[] { typeof(OsuModAutopilot), typeof(OsuModAutoplay), typeof(OsuModWiggle), typeof(OsuModTransform) };
+
+        private readonly HashSet<DrawableOsuHitObject> movingObjects = new HashSet<DrawableOsuHitObject>();
+        private int updateCounter;
+
+        public void Update(Playfield playfield)
+        {
+            // Avoid crowded judgment displays
+            playfield.DisplayJudgements.Value = false;
+
+            // Object destination updated when cursor updates
+            playfield.Cursor.ActiveCursor.OnUpdate += drawableCursor =>
```

Comment:
Subscribing to the cursor update every playfield update?? you probably want `IApplicableToDrawableRuleset` instead